### PR TITLE
user interfaces added. auth added. init done

### PR DESCRIPTION
**Pull request by blackkat98**
- User Interfaces added using Admin LTE 3 Templates
- Auth generated
- Login and Register Controllers not configured yet
- User and Country models not fixed yet

### CODE REVIEW

**Comment by blackkat98** [comment]
ready

**Comment by tuannt-0910** [comment]
1 commit / 1 pull

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -1,73 +1,75 @@
...
 
-                        <div class="form-group row mb-0">
-                            <div class="col-md-8 offset-md-4">
-                                <button type="submit" class="btn btn-primary">
-                                    {{ __('Login') }}
-                                </button>
+@section('content')
+<div class="card">
+    <div class="card-body login-card-body">
+        <p class="login-box-msg">Sign in to start your session</p>
```
Comment (tuannt-0910) [inline resources/views/auth/login.blade.php]: i18n

File: resources/views/auth/login.blade.php [hunk 2]
```diff
@@ -1,73 +1,75 @@
...
-                                        {{ __('Forgot Your Password?') }}
-                                    </a>
-                                @endif
-                            </div>
-                        </div>
-                    </form>
+        <form action="{{ route('login') }}" method="post">
+            {{ csrf_field() }}
+            <div class="input-group mb-3">
+                <input type="email" class="form-control @error('email') is-invalid @enderror" name="email" value="{{ old('email') }}" required autocomplete="email" autofocus placeholder="Email">
```
Comment (tuannt-0910) [inline resources/views/auth/login.blade.php]: i18n

File: resources/views/auth/login.blade.php [hunk 3]
```diff
@@ -1,73 +1,75 @@
...
+                    </div>
+                </div>
+                @error('email')
+                    <span class="invalid-feedback" role="alert">
+                        <strong>{{ $message }}</strong>
+                    </span>
+                @enderror
+            </div>
+            <div class="input-group mb-3">
+                <input type="password" class="form-control @error('password') is-invalid @enderror" name="password" required autocomplete="current-password" placeholder="Password">
```
Comment (tuannt-0910) [inline resources/views/auth/login.blade.php]: i18n

File: resources/views/auth/login.blade.php [hunk 4]
```diff
@@ -1,73 +1,75 @@
...
+                        <strong>{{ $message }}</strong>
+                    </span>
+                @enderror
             </div>
+            <div class="row">
+                <div class="col-8">
+                    <div class="icheck-primary">
+                        <input class="form-check-input" type="checkbox" name="remember" id="remember" {{ old('remember') ? 'checked' : '' }}>
+                        <label for="remember">
+                            Remember Me
```
Comment (tuannt-0910) [inline resources/views/auth/login.blade.php]: i18n

File: resources/views/auth/login.blade.php [hunk 5]
```diff
@@ -1,73 +1,75 @@
...
+                    <button type="submit" class="btn btn-primary btn-block btn-flat">Sign In</button>
+                </div>
+                <!-- /.col -->
+            </div>
+        </form>
+
+        <div class="social-auth-links text-center mb-3">
+            <p>- OR -</p>
+            <a href="#" class="btn btn-block btn-primary">
+                <i class="fab fa-facebook mr-2"></i> Sign in using Facebook
```
Comment (tuannt-0910) [inline resources/views/auth/login.blade.php]: i18n

File: resources/views/auth/login.blade.php [hunk 6]
```diff
@@ -1,73 +1,75 @@
...
+            </a>
+            <a href="#" class="btn btn-block btn-danger">
+                <i class="fab fa-google-plus mr-2"></i> Sign in using Google+
```
Comment (tuannt-0910) [inline resources/views/auth/login.blade.php]: i18n

File: resources/views/auth/login.blade.php [hunk 7]
```diff
@@ -1,73 +1,75 @@
...
+            </a>
         </div>
+        <p class="mb-1">
+            @if (Route::has('password.request'))
+                <a href="{{ route('password.request') }}">I forgot my password</a>
```
Comment (tuannt-0910) [inline resources/views/auth/login.blade.php]: i18n

File: resources/views/auth/login.blade.php [hunk 8]
```diff
@@ -1,73 +1,75 @@
...
+            @endif
+        </p>
+        <p class="mb-0">
+            <a href="{{ route('register') }}" class="text-center">Register a new membership</a>
```
Comment (tuannt-0910) [inline resources/views/auth/login.blade.php]: i18n

File: resources/views/auth/register.blade.php [hunk 9]
```diff
@@ -1,77 +1,120 @@
...
+        <p class="login-box-msg">Register a new membership</p>
 
-                            <div class="col-md-6">
-                                <input id="password-confirm" type="password" class="form-control" name="password_confirmation" required autocomplete="new-password">
-                            </div>
-                        </div>
+        <form action="{{ route('register') }}" method="post">
+            {{ csrf_field() }}
+            <div class="input-group mb-3">
+                <input type="text" class="form-control @error('name') is-invalid @enderror" name="name" value="{{ old('name') }}" required autocomplete="name" autofocus placeholder="Username">
```
Comment (tuannt-0910) [inline resources/views/auth/register.blade.php]: i18n

File: resources/views/auth/register.blade.php [hunk 10]
```diff
@@ -1,77 +1,120 @@
...
+                    </div>
+                </div>
+                @error('phone')
+                    <span class="invalid-feedback" role="alert">
+                        <strong>{{ $message }}</strong>
+                    </span>
+                @enderror
+            </div>
+            <div class="input-group mb-3">
+                <input type="password" class="form-control @error('password') is-invalid @enderror" name="password" required autocomplete="new-password" placeholder="Password">
```
Comment (tuannt-0910) [inline resources/views/auth/register.blade.php]: i18n

**Review by tuannt-0910 — COMMENTED** [review]
fix hết i18n trong pull này

**Pull request closed without merging**